### DISCUSSION TITLE
下载视频 以前的批量导出IDM链接文件功能在哪里？

### DISCUSSION THREAD

**Discussion by Lichtmann** [the1812/Bilibili-Evolved#3479] [提问 / Q&A] [answered]
以前批量下载视频时可以导出IDM链接文件， 现在怎么不行了？
![image](https://user-images.githubusercontent.com/29507184/177031606-985fbbe6-2339-4aba-9253-caaaada8c86d.png)
变成输出Toast
![image](https://user-images.githubusercontent.com/29507184/177031617-268f988a-8d2b-4167-9094-4f4aab0407ee.png)

**Comment by the1812** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/26504152/177031753-06ab965d-94e3-4d84-a4f6-2e8f6dfa40f1.png)

**Reply by Lichtmann** [reply]
同时我需要输出ffmpeg分P列表 .txt 用来为下载的flv改名

**Reply by the1812** [reply]
新版的应该自带文件名了, 不用再改名了
